src/js/components/ToggleButton.js

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import React from 'react';
+
+class ToggleButton extends React.Component {
+  render() {
+    let {
+      checkboxClassName,
+      checked,
+      children,
+      className,
+      id,
+      onChange
+    } = this.props;
+
+    return (
+      <span className={classNames(className)}>
+        <input
+          className={classNames(checkboxClassName)}
+          id={id}
+          name="checkbox"
+          onChange={onChange}
+          type="checkbox" />
+        <label
+          checked={checked}
+          htmlFor={id}>
+          {children}
+        </label>
+      </span>
+    );
+  }
+}
+
+ToggleButton.defaultProps = {
+  checked: false,
+  id: Date.now(),
+  onChange: function () {},
+
+  checkboxClassName: 'toggle-button'
+}
+
+ToggleButton.propTypes = {
+  checked: React.PropTypes.bool,
+  children: React.PropTypes.node,
+  id: React.PropTypes.oneOfType([
+    React.PropTypes.string,
+    React.PropTypes.number
+  ]),
+  onChange: React.PropTypes.func,
+
+  checkboxClassName: React.PropTypes.oneOfType([
+    React.PropTypes.string,
+    React.PropTypes.number,
+    React.PropTypes.array
+  ]),
+  className: React.PropTypes.oneOfType([
+    React.PropTypes.string,
+    React.PropTypes.number,
+    React.PropTypes.array
```

Comment:
We should probably add `React.PropTypes.object` for each of these class props since we're passing them into the `classnames` module anyway.